src/Proto.Mailbox/Mailbox.cs

```diff
@@ -187,10 +192,20 @@ public void Start()
             }
             catch (Exception e)
             {
-                await _invoker.EscalateFailureAsync(e, msg);
+                HandleProcessMessageException(e, msg);
             }
             return true;
         }
+        
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Honestly not sure what will happen, the JIT docs are very vague. `AggressiveInling` will apparently inline the code regardless of the method size, but it doesn't specify if that means immediately or only when it identifies it as a hot path. So I'm not sure if the JIT would eventually inline it, or if it decided that the method shouldn't/couldn't be inlined.

In any case though, actors shouldn't fail that often anyway, so I'm fine with leaving it as is. I doubt it will hurt performance in any meaningful way. I was mostly curious about whether the inlining will actually happen or not :)